### PR TITLE
Add run_exports

### LINKED ISSUE #53: add run_exports
Is there any reason why this package has no run_exports defined yet?

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #53

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,6 +36,10 @@ build:
   {% endif %}
   string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
 
+  run_exports:
+    # https://abi-laboratory.pro/index.php?view=timeline&l=netcdf-fortran
+    - {{ pin_subpackage('netcdf-fortran', max_pin='x.x') }} {{ build_pin }}
+
```
Comment (xylar) [inline recipe/meta.yaml:48]: @conda-forge/netcdf-fortran, does this seem right to all of you?

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipe/meta.yaml:48]: I'm following:
https://github.com/conda-forge/libnetcdf-feedstock/blob/7682e750c11aab024167d75a7c4830e74964d163/recipe/meta.yaml#L49-L57

**Comment by xylar** [comment]
@conda-forge-admin, please restart ci

**Comment by xylar** [comment]
@ocefpaf and @bekozi, this is ready to review if you have time.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:48]: I could not find info on the ABI laboratory. @isuruf what do you recommend?

Comment (isuruf) [reply recipe/meta.yaml:48]: x.x is fine according to the ABI laboratory URL above

Comment (ocefpaf) [reply recipe/meta.yaml:48]: Ouch! My dark theme hid that text as a faith grey and I could no read it. I guess that the oficial "bright" dark theme from GH is not that bad. I'll have to rever to it.

**Pull request merged**